### ISSUE TITLE
[Bug]: Randomizer flagged as Malware

### ISSUE THREAD

**Issue by CCReisz** [mint-choc-chip-skyblade/sshd-rando#358]
### Discord ID / GitHub Username

cc_reisz on discord

### What did you expect to happen?

I have run this software several times over the past few days, and generated multiple seeds.

### What actually happened?

AS of June 11th 2024, running the randomizer software causes Windows Defender to flag and quarantine the program. Defender detects it as "Trojan:Win32/Sabsik.EN.D!ml" If there is malicious code in the software, this should be corrected. If there is not, the software getting flagged like this makes using it more trouble than it's worth.

### OS

Windows

### Platform

Emulator

### Randomizer Version

1.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gymnast86** [comment]
Yeah, this has been a problem for a while even before we released. We submitted the software right before release for antivirus review and got back an email saying that it _was_ being incorrectly identified as malware in windows defender. If you update windows defender, it should hopefully fix the issue.
